### ISSUE TITLE
Make an embeddable nonprofit search component

### ISSUE THREAD

**Issue by osdiab** [everydotorg/donate-button#128]
We already have our Partner Search API working that allows for searching any nonprofit, powered by Constructor. We can make it super simple to use it by having an embeddable/`npm`-installable component that consumes it. https://www.notion.so/everydotorg/Partner-API-731e285581dc47efbe306c59c139d528

**Comment by osdiab** [comment]
cc @markulrich

**Comment by martinbianchi** [comment]
@osdiab Should we expose this component along with the Donation button and the widget? Or should we create a new package inside the repo?

**Comment by martinbianchi** [comment]
Also, should we follow some design guideline for this one? and which would be the final goal, I mean after the user searches and select a non-profit, what should happen?
And finally, would be great to have an api-key to test it while we develop the component

**Comment by osdiab** [comment]
Yep, I’m thinking these embeds could happen as sub packages here, using the same basic tech that the donate button does. Let’s hold off on actually implementing it until we set some designs for how it ought to look and spec it out a bit more, though, but should be ready to go soon. I’ll also give you a testing key shortly.
